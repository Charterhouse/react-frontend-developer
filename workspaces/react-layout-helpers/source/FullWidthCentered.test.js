import React from 'react'
import { render } from 'react-testing-library'
import { FullWidthCentered } from './'

describe('FullWidth', () => {
  it('has proper styling', () => {
    const { container } = render(<FullWidthCentered />)
    expect(container).toMatchSnapshot()
  })
})
