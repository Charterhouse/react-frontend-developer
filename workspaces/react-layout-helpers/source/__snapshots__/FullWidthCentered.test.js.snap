// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullWidth has proper styling 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column nowrap;
  -ms-flex-flow: column nowrap;
  flex-flow: column nowrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 20px;
  width: 100%;
  height: 100%;
  background-color: #f2f2f2;
  border-bottom: 1px solid #ff80c3;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
}

<div>
  <div
    class="emotion-0 emotion-1"
  />
</div>
`;
